modules/pipeline/providers/engine/worker.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package engine
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	"github.com/erda-project/erda/modules/pipeline/providers/leaderworker"
+	"github.com/erda-project/erda/modules/pipeline/providers/leaderworker/worker"
+)
+
+func (p *provider) reconcilePipeline(ctx context.Context, logicTask worker.Tasker) {
```

Comment:
confusion func name, I cannot distinct what different `reconcilePipeline` and `Reconcile`.

Reply:
adjust to `reconcileOnePipeline`.

`Reconcile` is a method of Reconciler provider, adjusted to `ReconcileOnePipelineUntilDone`.